api/AudioTrack.json

```diff
@@ -664,7 +664,7 @@
               "version_added": false
             },
             "ie": {
-              "version_added": false
+              "version_added": "11"
```

Comment:
api.VideoTrack.sourceBuffer should also change, but that need not happen in this PR, as it's not an inconsistency that jumps out at you when looking at any single MDN page.